src/librustdoc/html/static/storage.js

```diff
@@ -70,7 +70,7 @@ function usableLocalStorage() {
         return false;
     }
 
-    return true;
+    return window.localStorage !== null && typeof window !== 'undefined';
```

Comment:
What is the `typeof window !== 'undefined'` check for?

Reply:
Good catch! I'll update the code soon.